doc/classes/Viewport.xml

```diff
@@ -129,13 +129,23 @@
 		<method name="is_input_handled" qualifiers="const">
 			<return type="bool" />
 			<description>
+				Returns whether the current [InputEvent] has been handled. Input events are not handled until [method set_input_as_handled] has been called during the lifetime of an [InputEvent].
+				This is usually done as part of input handling methods like [method Node._input], [method Control._gui_input] or others, as well as in corresponding signal handlers.
+				If [member handle_input_locally] is set to [code]false[/code], this method will try finding the first parent viewport that is set to handle input locally, and return its value for [method is_input_handled] instead.
 			</description>
 		</method>
 		<method name="push_input">
 			<return type="void" />
 			<param index="0" name="event" type="InputEvent" />
 			<param index="1" name="in_local_coords" type="bool" default="false" />
 			<description>
+				Triggers the given [param event] in this [Viewport]. This can be used to pass an [InputEvent] between viewports, or to locally apply inputs that were sent over the network or saved to a file.
+				If [param in_local_coords] is set to [code]true[/code], the given [param event] will be transformed to the local coordinate space of the [Viewport].
```

Comment:
This formulation might be misleading. I suggest: "If [code]in_local_coords[/code] is [code]false[/code], the events position is in the embedders coordinates and will be converted to viewport coordinates. If [code]in_local_coords[/code] is [code]true[/code], the events position is in viewport coordinates."